x-pack/plugins/security_solution/public/timelines/components/flyout/bottom_bar/index.tsx

```diff
@@ -6,60 +6,19 @@
  */
 
 import { EuiPanel } from '@elastic/eui';
-import { rgba } from 'polished';
 import React from 'react';
 import styled from 'styled-components';
 
-import { IS_DRAGGING_CLASS_NAME } from '../../../../common/components/drag_and_drop/helpers';
 import { DataProvider } from '../../timeline/data_providers/data_provider';
 import { flattenIntoAndGroups } from '../../timeline/data_providers/helpers';
 import { DataProviders } from '../../timeline/data_providers';
-import { FLYOUT_BUTTON_BAR_CLASS_NAME, FLYOUT_BUTTON_CLASS_NAME } from '../../timeline/helpers';
+import { FLYOUT_BUTTON_BAR_CLASS_NAME } from '../../timeline/helpers';
 import { FlyoutHeaderPanel } from '../header';
 import { TimelineTabs } from '../../../../../common/types/timeline';
 
 export const getBadgeCount = (dataProviders: DataProvider[]): number =>
   flattenIntoAndGroups(dataProviders).reduce((total, group) => total + group.length, 0);
 
-const SHOW_HIDE_GLOBAL_TRANSLATE_Y = 50; // px
-const SHOW_HIDE_TIMELINE_TRANSLATE_Y = 0; // px
-
-const Container = styled.div.attrs<{ $isGlobal: boolean }>(({ $isGlobal = true }) => ({
```

Comment:
The template `bottomBar` handles a lot of this logic already, but any additional `timeline` specific logic was added to the styles seen in `template_wrapper/index`